SPECS-EXTENDED/hunspell/hunspell.spec

```
@@ -3,7 +3,7 @@
 Name:      hunspell
 Summary:   A spell checker and morphological analyzer library
 Version:   1.7.0
-Release:   6%{?dist}
+Release:   7%{?dist}
 # Source:  https://github.com/hunspell/hunspell/archive/refs/tags/v1.7.0.tar.gz
 Source:    https://github.com/hunspell/hunspell/archive/refs/tags/%{name}-%{version}.tar.gz
```

Weird formatting on this on that our cgmanifest check doesn't like- I'd go ahead and change it to the following:

```suggestion
Source0:  https://github.com/hunspell/hunspell/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

With a corresponding correction to the `hunspell` URL entry in the cgmanifest file.

---

thank you

---

A bit of historical context: the GitHub checks complaining about this line were introduced after most specs were created. For some it was easy to run a bulk fix but for others we have to rely on devs doing a one-time fix every time a spec is touched for the first time after the check got introduced. This is the latter case.;)

---

I think you'll also need to update the corresponding entry inside `cgmanifest.json` to `https://github.com/hunspell/hunspell/archive/refs/tags/v1.7.0.tar.gz`. You can run this to do it for you:

```bash
toolkit/scripts/update_cgmanifest.py last cgmanifest.json SPECS-EXTENDED/hunspell/hunspell.spec
```